contracts/colony/ColonyFunding.sol

```diff
@@ -218,6 +218,7 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs { // ignore-swc-123
   )
   public
   stoppable
+  domainNotDeprecated(getDomainFromFundingPot(_toPot))
```

Comment:
I'd say not much because even in the three calls, at least two are for different pots. So it's really only one duplicate call, and I'm not sure it's worth getting rid of the convenience/cleanness of the modifier to move it into a function / store the result as a variable.